it/uk/gov/hmrc/traderservices/controllers/TraderServicesFrontendISpec.scala

```diff
@@ -188,32 +188,32 @@ class TraderServicesFrontendISpec
         )
       }
 
-      "submit the form and, if Hold, ask next does the consignment has any priority goods" in {
-        implicit val journeyId: JourneyId = JourneyId()
-        journey.setState(
-          AnswerExportQuestionsRequestType(
-            ExportQuestionsStateModel(
-              DeclarationDetails(EPU(235), EntryNumber("A11111X"), LocalDate.parse("2020-09-23")),
-              ExportQuestions()
-            )
-          )
-        )
-        givenAuthorisedForEnrolment(Enrolment("HMRC-XYZ", "EORINumber", "foo"))
-
-        val payload = Map("requestType" -> "Hold")
-
-        val result = await(request("/pre-clearance/export-questions/request-type").post(payload))
-
-        result.status shouldBe 200
-        result.body should include(htmlEscapedMessage("view.export-questions.hasPriorityGoods.title"))
-        result.body should include(htmlEscapedMessage("view.export-questions.hasPriorityGoods.heading"))
-        journey.getState shouldBe AnswerExportQuestionsHasPriorityGoods(
-          ExportQuestionsStateModel(
-            DeclarationDetails(EPU(235), EntryNumber("A11111X"), LocalDate.parse("2020-09-23")),
-            ExportQuestions(requestType = Some(ExportRequestType.Hold))
-          )
-        )
-      }
+//      "submit the form and, if Hold, ask next does the consignment has any priority goods" in {
```

Comment:
This page is largely fine and the commented tests seem safe to be removed from what I can see. Some minor little things though we can change mostly for readability